java/activator/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             activator
+version          1.3.12
+categories       java devel
+platforms        darwin
+supported_archs  noarch
+license          Apache-2
+maintainers      gmail.com:nhoj.patrick \
+                 openmaintainer
+
+description      TypeSafe activator framework used by the java web play framework
+
+long_description \
+    Play Framework makes it easy to build web applications with Java & Scala. \
+    Play is based on a lightweight, stateless, web-friendly architecture. \
+    Built on Akka, Play provides predictable and minimal resource consumption \
+    (CPU, memory, threads) for highly-scalable applications.
+
+homepage         https://www.playframework.com
+
+master_sites     https://downloads.typesafe.com/typesafe-activator/${version}/
+distname         typesafe-${name}-${version}
+worksrcdir       ${name}-dist-${version}
+
+checksums        rmd160 22b6dff199f48e7cb59e24b6afd6c8d3a2a3f373 \
+                 sha256 3740355ce4d4c099b9349704dc5851341e731981b25094b7b7dcefe7eceff530
+
+extract.mkdir    yes
+extract.suffix   .zip
```

Comment:
Please remove those two lines. The `use_zip` already switches to zip and I guess that `extract.mkdir yes` results in files being extracted to `work/activator-dist-1.3.12/activator-dist-1.3.12`.